src/js/components/Card/stories/Simple.js

```diff
@@ -1,42 +1,34 @@
 import React from 'react';
 
-import {
-  grommet,
-  Box,
-  Card,
-  CardBody,
-  CardFooter,
-  CardHeader,
-  Grommet,
-  Text,
-} from 'grommet';
+import { Box, Card, CardBody, CardFooter, CardHeader, Text } from 'grommet';
 
 export const Simple = () => (
-  <Grommet theme={grommet}>
-    <Box pad="large" gap="medium" height="large" width="medium">
-      <Card pad="small" background="dark-1" gap="medium">
-        <CardHeader>header</CardHeader>
-        <CardBody>body</CardBody>
-        <CardFooter>footer</CardFooter>
-      </Card>
-      {/* internal components are optional  */}
-      <Card pad="small" gap="medium" background="light-4">
-        <CardBody>body</CardBody>
-        <Box>box - random component</Box>
-      </Card>
-      {/* internal components order is flexible  */}
-      <Card pad="small" gap="medium">
-        <CardBody>body</CardBody>
-        <CardHeader>header</CardHeader>
-        <CardFooter>footer</CardFooter>
-      </Card>
-      {/* children that are not Card internal components */}
-      <Card pad="small" gap="medium" background="light-1">
-        <Text>text - random component</Text>
-        <Box>box - random component</Box>
-      </Card>
-    </Box>
-  </Grommet>
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Box pad="large" gap="medium" height="large" width="medium">
```

Comment:
```suggestion
  <Box pad="large" gap="medium" width="medium">
```
Think we should remove this fixed height and let the cards take the vertical height that they need.

Reply:
This should resolve the issue we see on the story when using hpe theme.

Reply:
Solved the problem. Thank you